### ISSUE TITLE
vm-virtio: FR: support for VIRTIO_F_IN_ORDER and Packed Ring

### ISSUE THREAD

**Issue by aftersnow** [rust-vmm/vm-virtio#241]
These two features are benefits to performance

**Comment by aftersnow** [comment]
> I don't think this crate needs to care about `VIRTIO_F_IN_ORDER` - that's more for the VMM to do it.

Yes, thanks for reply. The "Packed Ring" should benefits for performance, do you have plan to add support for it?

**Comment by uran0sH** [comment]
I will work on adding support for packed vring if no one works on it.

**Comment by stefano-garzarella** [comment]
@uran0sH that would be nice! If it can help, we implemented packed virtqueues in `libblkio`, but we didn't use rust-vmm crates in that project. You can find the code here: https://gitlab.com/libblkio/libblkio/-/blob/main/virtio-driver/src/virtqueue/packed.rs?ref_type=heads

**Comment by stefano-garzarella** [comment]
Done in https://github.com/rust-vmm/vm-virtio/pull/310
